### ISSUE TITLE
format on save doesn't show error messages anymore

### ISSUE THREAD

**Issue by faassen** [jlongster/prettier-atom#54]
Previously if I had a syntax error in my code prettier would show me an error popup in red that indicated what I've been doing wrong.

But a few updates ago it stopped doing this. I noticed the new "On format on Save, don't show error message" option, which is disabled. It's absent in config.cson.

I do see the syntax errors in the development console.

I really miss the error messages.

**Comment by robwise** [comment]
@faassen You should still be getting errors on the latest build. Have you updated your package to the latest version? 

Here is how the feature works:

![demo of prettier error message suppression](https://cl.ly/1D0M0h241A3a/Screen%20Recording%202017-02-21%20at%2005.51%20AM.gif)

**Comment by faassen** [comment]
Yes, I am on 0.18.0. "Format on save" is on, "On format on Save, don't show error message" is off. 

If I don't introduce syntax errors in my code it formats as before. But if I have a syntax error, I don't get the red popup anymore. It used to work as you demonstrate for me. Sometime last week, it stopped working.

I can see the syntax error reported in the Atom dev tool console, but that's it. This is what's in the dev tools when I save (helpers.js line 29):

```
Error executing Prettier: SyntaxError: Unexpected token, expected { (1:10)
> 1 | class Foo function bar {
    |           ^
  2 | 
  3 | fadf
  4 |   fadfjdflkj
    at Parser.pp$5.raise 
```

But I don't get any popup outside of the dev tools.

**Comment by robwise** [comment]
Doh! I hadn't released the fix for this (#53), my apologies. I just published it.

**Comment by garth** [comment]
Now there is no way to turn error on save off, checked or unchecked the error is always displayed.
